### PR TITLE
AC Automated Test - LRAC-8498 | master

### PR DESCRIPTION
**Pull request by MathausC**
https://issues.liferay.com/browse/LRAC-8498

### CODE REVIEW

**Comment by MathausC** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCriteriaOverview.testcase [hunk 1]
```diff
@@ -31,12 +31,85 @@ definition {
...
+
+			ACSegments.editWebBehaviorCriterion(
+				indexField = "15",
+				searchTerm = "To See Blog");
+
+			ACSegments.saveSegment();
+		}
+
+		task ("Click View All Criteria, ascroll down and see the last criteria.") {
+			ACUtils.clickAnyButton(button = "View All Criteria");
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCriteriaOverview.testcase:107]: typo: ascroll -> scroll

**Comment by alexcandidos** [comment]
Hi @MathausC , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**